src/pages/home/report/ReportActionsView.js

```diff
@@ -197,6 +205,11 @@ class ReportActionsView extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        // Update last read action when Report data is loaded
```

Comment:
Can we change this comment to 

>Update the last read action for the report currently in view when report data finishes loading. This report should now be up-to-date and since it is in view we mark it as read.

Reply:
ok